liquidcore/home/discover.py

```diff
@@ -0,0 +1,20 @@
+import requests
+from django.conf import settings
+from requests.exceptions import ConnectionError
+from json.decoder import JSONDecodeError
+
+def get_status():
+    if not settings.DISCOVERY_URL:
+        raise RuntimeError("settings.DISCOVERY_URL was not configured")
+    try:
+        response = requests.get(settings.DISCOVERY_URL).json()
+        response.setdefault("running", True)
```

Comment:
Well, the `running` flag seems to be the responsibility of this bit of code, not upstream, so there should definitely be `"running": True` here.